Source/ActivityPub.Types/AS/ASActivity.cs

```diff
@@ -92,11 +103,23 @@ public LinkableList<ASObject> Actor
     ///     For instance, if a particular action results in the creation of a new resource, the result property can be used to describe that new resource.
     /// </summary>
     /// <seealso href="https://www.w3.org/TR/activitystreams-vocabulary/#dfn-result" />
-    public Linkable<ASObject>? Result
+    public LinkableList<ASObject>? Result
     {
         get => Entity.Result;
         set => Entity.Result = value;
     }
+
+    /// <summary>
+    /// Describes the indirect object, or target, of the activity. The precise meaning of the target is largely
+    /// dependent on the type of action being described but will often be the object of the English preposition "to".
+    /// For instance, in the activity "John added a movie to his wishlist", the target of the activity is John's
+    /// wishlist. An activity can have more than one target. 
+    /// </summary>
+    public LinkableList<ASObject>? Target
```

Comment:
this one exists in `ASTargetedActivity` for the same reason as `Object`. It's somewhat convoluted, but this protects against null objects or targets when they might be required. It also ensures that they *are* null for those activity types that prohibit them.

Reply:
flattened in ASActivity